src/classlibnative/bcltype/system.cpp

```diff
@@ -73,9 +73,11 @@ FCIMPLEND;
 
 
 
-#ifndef FEATURE_CORECLR
 INT64 QCALLTYPE SystemNative::GetWorkingSet()
 {
+#if PLATFORM_UNIX
+    return 0; // TODO: Implement in PAL
```

Comment:
A similar question came up when we started implementation of `System.Private.CoreLib.Native` PAL for CoreRT. I was not happy that `System.Private.CoreLib.Native` is on path to duplicate a bunch of methods from `System.Native`.